client/scss/core.scss

```diff
@@ -140,6 +138,8 @@ These are classes for components.
 @import 'components/comments-notification-dropdown';
 @import 'components/bulk_actions';
 
+@import '../src/components/Sidebar/Sidebar';
```

Comment:
I had to move this to the bottom because there are button styles that conflict with it.